src/t8_schemes/t8_default/t8_default_common/t8_default_common.hxx

```diff
@@ -65,7 +207,12 @@ class t8_default_scheme_common_c: public t8_eclass_scheme_c {
    * children.
    */
   t8_gloidx_t
```

Comment:
Please add a comment that pyramids have their own implementation of this function.